angrmanagement/ui/views/interaction_view.py

```diff
@@ -295,6 +295,7 @@ def _start_interaction(self):
             req_msg = 'To use this feature you need to install the following:\n\n\t' + '\n\t'.join(is_missing)
             req_msg += '\n\nInstall them to enable this functionality.'
             req_msg += '\nRelaunch angr-management after install.'
+            req_msg += '\nNote: This cannot be done for pyinstaller builds.'
```

Comment:
I don't like this way of handling this. If we are in a frozen build and a feature is not available, we should just not show that feature. If for some reason we must show that the feature is there or disabled, the message should be more along the lines of "This feature is not available in this version of angr management".

Reply:
I agree; that's why I made: https://github.com/angr/angr-management/issues/844
I'd like this soft-dependency stuff gone. But that's for a different PR.

Reply:
The interaction view might be able to go away soon, but archr definitely won't. If we can improve test quality and get it to work cross-platform it might be a hard dependency, but that's not realistically going to happen any time soon. Checking if it is a pyinstaller build should be easy enough and we can just add a switch to the error message. If `getattr(sys, "frozen", None)` returns something other than `None`, then we know we're in pyinstaller.